### DISCUSSION TITLE
Activity Bar Selections Do Not Appear

### DISCUSSION THREAD

**Discussion by dtsmith2001** [coder/code-server#3389] [Q&A] [answered]
I've searched the Visual Studio Code help, code-server issues, and the keyboard shortcuts. I've also tried an upgrade of code-server and an uninstall/reinstall.

My activity bar selections at the left have disappeared. I am able to bring up individual selections using shift-control-p and search for it. But I do not have the full activity bar so I can click and bring up the various parts - Extensions, Files, Menu, etc.

I am using Brave Version 1.24.84 Chromium: 90.0.4430.93 (Official Build) (64-bit).

Has anyone else had this problem?

Is there any other information I should provide?

![Screenshot](https://user-images.githubusercontent.com/3189861/118273803-648b2e80-b492-11eb-82bb-b6d2cd849102.png)

**Comment by ghost** [comment, marked as answer]
Try `Ctrl + Shift + P` > something like `Toggle activity bar`. If this works then make a keybind for toggling,

**Reply by jsjoeio** [reply]
I can confirm — this works for me! Here's a video:

https://user-images.githubusercontent.com/3806031/118301664-2ee94400-b498-11eb-96df-12a73c34551c.mov

**Reply by MxD-js** [reply]
Works for me also.

**Reply by dtsmith2001** [reply]
Thanks so much.

**Comment by dtsmith2001** [comment]
Thank you so much

------------------------------------------------------------------------

Dale Smith, Ph.D. ***@***.***>
All times are Eastern
